packages/bpk-component-nudger/src/BpkNudger.module.scss

```diff
@@ -67,3 +67,8 @@
     }
   }
 }
+
+[dir='rtl'] .bpk-nudger {
```

Comment:
There's a [bpk-rtl](https://github.com/Skyscanner/backpack/blob/55c6d173303b689029a78198223ee3322bda4661/packages/bpk-mixins/src/mixins/_utils.scss#L133) mixin we should probably use